CocoaPods/macosx-10.12/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module CSQLite [system] {
+    header "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/usr/include/sqlite3.h"
```

Comment:
As above, this causes build errors for all users who don't have their copy of Xcode 8.x at that path.